util/generate_man.js

```diff
@@ -72,7 +71,7 @@ const processDovecotMdPost = () => {
 		})
 
 		visit(parents(tree), 'defListDescription', function (node, index, parent) {
-			/* Convert the actual description text to a blockquote, so
+			/* Convert the actsual description text to a blockquote, so
```

Comment:
typo created

Reply:
that's not actually my typo, but i'll fix it.